src/NServiceBus.AcceptanceTests/Forwarding/When_forwarding_is_configured_for_endpoint.cs

```diff
@@ -1,36 +1,49 @@
 ﻿namespace NServiceBus.AcceptanceTests.Forwarding
 {
-    using System;
+    using System.Collections.Generic;
     using System.Threading.Tasks;
     using AcceptanceTesting;
     using EndpointTemplates;
     using NUnit.Framework;
 
-    public class When_ForwardReceivedMessagesTo_is_set : NServiceBusAcceptanceTest
+    public class When_forwarding_is_configured_for_endpoint : NServiceBusAcceptanceTest
     {
         [Test]
         public async Task Should_forward_message()
         {
             var context = await Scenario.Define<Context>()
-                .WithEndpoint<EndpointThatForwards>(b => b.When((session, c) => session.SendLocal(new MessageToForward())))
+                .WithEndpoint<EndpointThatForwards>(b => b.When((session, c) =>
+                {
+                    var options = new SendOptions();
+
+                    options.RouteToThisEndpoint();
+                    options.RouteReplyTo(CustomReplyToAddress);
```

Comment:
Good catch, was first testing only the reply to address header but then switched to checking "all the things" and forgot to revert the code, will fix